wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift

```diff
@@ -233,11 +233,11 @@ final class ProfileHeaderViewController: UIViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
+        userStatus.isVerified = isSelfUserProteusVerifiedUseCase.invoke()
         Task {
+            defer { updateNameLabel() }
```

Comment:
Why `defer` and not just write the line at the end after `do-catch`? For me `defer` is always a little brain breaker, because the code is not executed the obvious order..

Reply:
Done.